src/app/features/issue/providers/azuredevops/azuredevops.const.ts

```diff
@@ -0,0 +1,93 @@
+// TODO use as a checklist
+import { AzuredevopsCfg } from './azuredevops.model';
+import { T } from '../../../../t.const';
+import {
+  ConfigFormSection,
+  LimitedFormlyFieldConfig,
+} from '../../../config/global-config.model';
+
+export const DEFAULT_AZUREDEVOPS_CFG: AzuredevopsCfg = {
+  isEnabled: false,
+  organization: null,
+  project: null,
+  token: null,
+  isSearchIssuesFromAzuredevops: false,
+  isAutoPoll: false,
+  isAutoAddToBacklog: false,
+  filterUsername: null,
+};
+
+export const AZUREDEVOPS_POLL_INTERVAL = 10 * 60 * 1000;
+export const AZUREDEVOPS_INITIAL_POLL_DELAY = 8 * 1000;
+
+export const AZUREDEVOPS_API_BASE_URL = 'https://dev.azure.com';
```

Comment:
Just thought of this integration when discussing in my team, awesome that you are on it! Will definitely try this out. One request: can you please make this URL configurable? My company uses on-premise Azure DevOps Server.